packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1627,6 +1628,24 @@ async function handler(
       };
     }
   }
+  if (
+    Object.prototype.hasOwnProperty.call(eventType, "bookingLimits") ||
+    Object.prototype.hasOwnProperty.call(eventType, "durationLimits")
+  ) {
+    const startAsDate = dayjs(reqBody.start).toDate();
+    if (eventType.bookingLimits) {
+      await checkBookingLimits(
+        eventType.bookingLimits as IntervalLimit,
+        startAsDate,
+        eventType.id,
+        eventType.schedule?.timeZone
+      );
+    }
+    if (eventType.durationLimits) {
+      await checkDurationLimits(eventType.durationLimits as IntervalLimit, startAsDate, eventType.id);
+    }
+  }
+
```

Comment:
Why are we adding this code here again (looks same as https://github.com/calcom/cal.com/pull/13526/files#diff-a7c21727d9a0d4d1aaff9d24fca9293bc28468861bc8022dfa03116908b6fae7R1124)? And aren't we now calling `checkBookingLimit` and `checkDurationLimits` twice for non-seated events?

Reply:
Originally I placed the code here for the time being (performing the limits check on seated event was just blocking the logic) and It was and is open for discussion on how to structure this.

A quick fix will be place a condition such as `eventType.seatsPerTimeSlot` but i don't feel good about it. What do you suggest?

```
  if (
    eventType.seatsPerTimeSlot &&
    (Object.prototype.hasOwnProperty.call(eventType, "bookingLimits") ||
    Object.prototype.hasOwnProperty.call(eventType, "durationLimits"))
  ) { }
```